jwst_reffiles/bad_pixel_mask/bad_pixel_mask.py

```diff
@@ -726,6 +728,35 @@ def get_fastaxis(filename):
     return fastaxis, slowaxis
 
 
+def get_max_dead_norm_signal_default(instrument, detector):
+    """
+    Finds the default max_dead_norm_signal value to use for the 
+    absolute_rate dead pixel search type.
+
+    Parameters
+    ----------
+    instrument : str
+        The name of the instrument
+    
+    detector : str
+        The name of the detector
+
+    Returns
+    -------
+    default_value : float
+        The default max_dead_norm_signal value
+    """
+    if instrument == 'NIRCAM':
+        detectors = ['NRCA1', 'NRCA2', 'NRCA3', 'NRCA4', 'NRCALONG', 
+                     'NRCB1', 'NRCB2', 'NRCB3', 'NRCB4', 'NRCBLONG']
+        defaults = [25.0, 25.0, 30.0, 30.0, 40.0, 
```

Comment:
Would be good to put a check in here so that these only apply if there is no normalization being done.

Reply:
added condition to only use these values if no normalization is done; otherwise, it just uses the original 0.05 default